docs/dev/style-guide.rst

```diff
@@ -67,3 +71,98 @@ but even more importantly,
 it helps and includes readers by giving them quick and easy access to terms that they may be unfamiliar with.
 
 Use an external link or Intersphinx reference when a term is clearly defined elsewhere.
+
+Cross-references
+----------------
+
+We like to cross-reference other articles with a definition list inside a ``seealso::`` admonition box.
+It looks like this:
+
+.. code-block:: rst
+
+   .. seealso::
+
+      :doc:`/other/documentation/article`
+        You can learn more about <concept> in this (how-to/description/section/article)
+
+Diátaxis Framework
+------------------
+
+We apply the methodology and concepts of the Diátaxis Framework.
+This means that *both content and navigation path* for all sections should fit a single category of the 4 Diátaxis categories:
+
+* Tutorial
+* Explanation
+* How-to
+* Reference
+
+.. seealso::
+
+   `https://diataxis.fr/ <https://diataxis.fr/>`__
+     The official website of Diátaxis is the main resource.
+     It's best to check this out before guessing what the 4 categories mean.
+
+.. warning:: **Avoid minimal changes**
+
+   If your change has a high coherence with another proposed or planned change,
+   propose the changes in the same PR.
+
+   By multi-tasking on several articles about the same topic,
+   such as an explanation *and* a how-to,
+   you can easily design your content to end up in the right place *Diátaxis-wise*.
+   This is great for the author and the reviewers
+   and it saves coordination work.
+
+   Minimal or isolated changes generally raise more questions and concerns
+   than changes that seek to address a larger perspective.
+
+Explanation
+^^^^^^^^^^^
```

Comment:
I think we generally use `~` for third-level headings... should probably go in the style guide :D

Reply:
I really don't get Sphinx/reST on this particular thing. You can apparently choose the heading characters and then the parser just magically understands. It made me completely unable to memorize which character I'm supposed to use.

Reply:
Can you outline what you believe is our convention for outlining headers?

```
Header 1
======== 

Header 2
--------

Header 3
~~~~~~~~

Header 4
^^^^^^^^
```

?

Reply:
reST is funny. You can use almost any symbol character you want, including some unicode I believe.

```
Heading 1
)))))))))

Heading 2
(((((((((

Heading 3
@@@@@@@@@

Heading 4
!!!!!!!!!
```

Heading characters just need to be relative to the first heading character you used and docutils is happy :upside_down_face:

Reply:
Yea, that's also what Plone uses: https://docs.plone.org/about/contributing/rst-styleguide.html#headings

Reply:
Do we want to do the Plone version? It's not fully consistent with most of our articles, since we usually don't have this:

```
========
Header 1
========
```

But I like the Plone version, personally I have the easiest time remembering that order... like you say `=, -, ~, ^` feels somehow like how the characters should be ordered :)

Reply:
Fixed the headlines and added a new section about headlines :+1: